cli/lsp/tsc.rs

```diff
@@ -4496,6 +4496,7 @@ impl<'a> ToV8<'a> for TscRequestArray {
 
     let method_name = deno_core::FastString::from_static(method_name)
       .v8_string(scope)
+      .unwrap()
```

Comment:
Ugh, I must have missed that. Why is it now a `Result`/`Option`?

Reply:
https://github.com/denoland/deno_core/commit/0ff818e78e4899d7d93a78743222323ed1c0b580